src/authutil.ts

```diff
@@ -74,13 +74,11 @@ function writeFeedToFile(
             i < json.configuration.packageSources.add.length;
             i++
           ) {
-            core.debug(json.configuration.packageSources.add[i]);

```

Comment:
core.debug( OBJECT ) doesnt work so i rearranged the code a little bit here to pass strings to core.debug